pkgs/development/libraries/fontconfig/make-fonts-conf.xsl

```diff
@@ -28,8 +28,6 @@
       <!-- /var/cache/fontconfig is useful for non-nixos systems -->
       <cachedir>/var/cache/fontconfig</cachedir>
 
-      <!-- fontconfig distribution conf.d -->
-      <include><xsl:value-of select="$fontconfig" />/etc/fonts/conf.d</include>
```

Comment:
For reference, the above line is now done by lines 167 through 171 in `fontconfig.nix`: https://github.com/ericsagnes/nixpkgs/blob/da64fed0a378e243b3c50847308a29fd9667810e/nixos/modules/config/fonts/fontconfig.nix#L167-L171